platform/ios/src/MGLFaux3DUserLocationAnnotationView.h

```diff
@@ -1,7 +1,12 @@
 #import <UIKit/UIKit.h>
 #import "MGLUserLocationAnnotationView.h"
 
+const CGFloat MGLUserLocationAnnotationDotSize = 22.0;
+const CGFloat MGLUserLocationAnnotationHaloSize = 115.0;
+
+const CGFloat MGLUserLocationAnnotationPuckSize = 45.0;
+const CGFloat MGLUserLocationAnnotationArrowSize = MGLUserLocationAnnotationPuckSize * 0.6;
```

Comment:
Are these constants used anywhere outside of MGLFaux3DUserLocationAnnotationView.m? If not, I think they can stay in that file.

Reply:
They were during the initial refactor, but I ended up trying to avoid using these outside of `MGLFaux3DUserLocationAnnotationView.m`. I’ll see about moving these back to there.

Reply:
`MGLUserLocationHeadingBeamLayer` still needs `MGLUserLocationAnnotationHaloSize`, which unfortunately can’t be derived from the user location annotation itself.